pages/hub/insights/index.tsx

```diff
@@ -42,69 +56,155 @@ const InsightsHub: WithPageLayout = () => {
-    <div className="container flex flex-col w-full gap-4 py-2">
-      <div className="justify-between block py-2 sm:flex ">
-        <div>
-          <Title className="!text-2xl !leading-none !font-medium  mb-4" level={1}>
-            Insights
-          </Title>
-          <Text className="my-8">
-            Welcome to your Insights Hub! Here, you can set up pages to view all of your insights or other open source
-            insights in one place.
-          </Text>
-        </div>
-        <div className="flex items-center gap-3 mt-4">
-          {/* Search box temporarily hidden */}
-          <div className="hidden w-58">
-            <Search placeholder="Search repositories" className="max-w-full" name={"query"} />
+    <Tabs value={router.pathname.split("/")[2] as TabKey} onValueChange={onTabChange}>
+      <div className="container flex flex-col w-full gap-4 py-2">
+        <Title className="-mb-6 text-base text-sauced-orange">Your pages</Title>
+        <div className="items-center justify-between block py-2 sm:flex ">
+          <TabsList className="gap-3">
+            {(Object.keys(tabs) as TabKey[]).map((tab) => (
+              <TabsTrigger
+                asChild
+                key={tab}
+                className="!px-0 !justify-start data-[state=active]:text-black text-gray-300/90 cursor-pointer"
+                value={tab}
+              >
+                <Title className={clsx("!text-3xl !leading-none !font-medium mx-0")} level={1}>
+                  {tabs[tab]}
+                </Title>
+              </TabsTrigger>
+            ))}
+          </TabsList>
+          <div className="flex items-center gap-3 mt-4">
+            {/* Search box temporarily hidden */}
+            <div className="hidden w-58">
+              <Search placeholder="Search repositories" className="max-w-full" name={"query"} />
+            </div>
+            {router.pathname.split("/")[2] === "insights" ? (
+              <Link href={"/hub/insights/new"}>
+                <Button variant="primary">Add Insight Page</Button>
+              </Link>
+            ) : (
+              <Link href={"/hub/lists/new"}>
+                <Button variant="primary">Add List</Button>
+              </Link>
+            )}
           </div>
-          <Link href={"/hub/insights/new"}>
-            <Button variant="primary">Add Insight Page</Button>
-          </Link>
         </div>
+
+        <TabsContent value={"insights" satisfies TabKey}>
+          <InsightsTab {...user} />
+        </TabsContent>
+        <TabsContent value={"lists" satisfies TabKey}>
+          <ListTab />
+        </TabsContent>
+
+        <Link
+          passHref
+          href={"/hub/insights/new"}
+          className="w-full py-5 text-lg text-center border rounded-lg bg-light-slate-4 text-light-slate-11 md:py-8 lg:py-10 border-light-slate-7"
+        >
+          {router.pathname.split("/")[2] === "insights" ? "Create a new Insight Page" : "Create a new List"}
```

Comment:
This should route to the `NewListCreationPage`? When I click this, it sends me to the new insights page

Reply:
Oops... missed that 😄.

updated now to the correct link, but the actual page will be created at https://github.com/open-sauced/app/pull/1698